.travis.yml

```diff
@@ -42,8 +42,8 @@ env:
     - WP_ADMIN_PASSWORD="password"
 
 before_install:
-  # Rollback Composer to version 1.
-  - composer self-update --rollback
+  # Set Composer to version 2.
```

Comment:
This looks like a good chance to move Common build away from Travis CI, as is the case for the other plugins, and just remove any configuration file related to it.